docs/source/configuration/authn-jwt.mdx

```diff
@@ -4,6 +4,12 @@ subtitle: Restrict access to credentialed users and systems
 description:  Protect sensitive data by enabling JWT authentication in the Apollo GraphOS Router. Restrict access to credentialed users and systems.
 ---
 
+<SummitCallout
```

Comment:
`<SummitCallout>` and `<PremiumFeature>` have different icon + text layout, and they don't look consistent together [here](https://deploy-preview-5830--apollo-router-docs.netlify.app/configuration/authn-jwt). Move the summit icon?

Reply:
The <SummitCallout> follows the styling of Tips/Cautions/etc.

See below on the same page:

<img width="793" alt="image" src="https://github.com/user-attachments/assets/58196ef8-5f3e-4c63-8eee-b545caafe622">


I'm personally fine with it, but we can definitely try to align all callout styles in a different PR!